src/components/PopupWithForm.js

```diff
@@ -16,15 +17,27 @@ export default class PopupWithForm extends Popup {
     return formValues;
   }
 
-  // closePopup() {
-  //   super.closePopup();
-  // }
+//  openPopup() {
+//   this._saveButton.textContent = "Save";
+//   super.openPopup();
+//  }
+
+ buttonTextSave () {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do. **Please, correct it everywhere in the code**